src/features/claims/claims-list/ClaimListFilters.tsx

```diff
@@ -33,6 +41,33 @@ const FilterElement = styled.div`
   }
 `
 
+const StyledLabel = styled(Label)`
+  display: flex;
+  align-items: center;
+
+  & span {
+    margin-right: 1em;
+  }
+
+  & svg {
+    width: 15px;
+    height: 15px;
+
+    &:hover {
+      cursor: help;
+    }
+  }
+`
+
+const LabelWithPropower = ({ label, propover }) => (
```

Comment:
Typo 😅 

```suggestion
const LabelWithPopover = ({ label, popover }) => (
```

Also, it always makes me uncomfortable with untyped arguments - can we add that they're strings hehe? 😄